app/views/directives/edit-lifecycle-hook.html

```diff
@@ -10,24 +10,22 @@
 
       <div class="form-group">
         <label for="actionType" class="required">Lifecycle Action</label><br/>
-        <div class="radio">
-          <label class="radio-inline">
-            <input type="radio"
-                    name="{{type}}-action-newpod"
-                    ng-model="action.type"
-                    value="execNewPod"
-                    aria-describedby="action-help">
-            Run a specific command in a new pod
-          </label>
-          <label class="radio-inline">
-            <input type="radio"
-                  name="{{type}}-action-images"
-                  ng-model="action.type"
-                  value="tagImages"
-                  aria-describedby="action-help">
-              Tag image if the deployment succeeds
-          </label>
-        </div>
+        <label class="radio-inline">
+          <input type="radio"
+          name="{{type}}-action-newpod"
+          ng-model="action.type"
+          value="execNewPod"
+          aria-describedby="action-help">
```

Comment:
Sorry, I'm picking nits, but can you indent the attributes as before? I find it harder to read this way.